src/components/Heading/Heading.mdx

```diff
@@ -145,6 +146,28 @@ to `level="one"`.
 
 <Canvas of={HeadingStories.SizeStylesDeprecated} />
 
+## Overline and Subtitle
+
+The `Heading` component offers two additional text elements as part of the full
+heading lockup: overline and subtitle. When the `overline` or `subtitle` props
+are passed, the `Heading` component will handle the styling for the
+corresponding text element.
+
+Use the `overline` prop to set the text for an eyebrow text element that sits
```

Comment:
Per @EdwinGuzman?
Perhaps mention that the text for overline will always be in uppercase.